airbyte-integrations/connectors/source-snowflake/metadata.yaml

```diff
@@ -1,7 +1,7 @@
 data:
   allowedHosts:
     hosts:
-      - ${host#https://}
+      - "*. snowflakecomputing.com"
```

Comment:
There is an extra space, i believe that should be there. Like this:
- `"*.snowflakecomputing.com"`

Reply:
The white-space is not needed, and it's not allowed either for the snowflake domain.